packages/headless/src/api/search/search/search-request.ts

```diff
@@ -59,3 +59,26 @@ export type SearchRequest = BaseParam &
   AnalyticsParam &
   ExcerptLength &
   ActionsHistoryParam;
+
+// Change this list when changing the fields exposed by `FieldsToIncludeParam`
+export const SearchDefaultFieldsToInclude: string[] = [
```

Comment:
It might be a bit more complicated than that.

There's a partial overlap in `fields-state.ts`, where we already have a list of the bare minimum fields that are needed in order for various analytics events to work properly. These should be considered "mandatory in all scenarios". 

Then you have a couple of them here which are really useful to power any result list (date, parents), but not strictly mandatory for all use cases.

Then all those `ec_${something}` fields are commerce specific, and we would only need them when dealing with commerce pages. So they should not have been put in `atomic-result-list` in the first place, in my opinion.

So... all in all, just to say I'm not 100% certain how we should subdivide that list to ensure that there's just the strict minimum requested depending on the use case (ie: which type of page the end user is trying to build).

If we request too much all the time, it will work correctly, but: 

* In the best case, the worst that will happen is that it pollute the request with useless fields that are absent from the index anyway.

* In the worst case, this will have negative performance impact.

So yeah, I'm not sure how we should arrange this... maybe the simplest would be various exported list with explicit naming that describe when they should be used ? 🤔

Reply:
I think it makes sense to have a 'bare minimum' and then a couple of others like 'defaults for ecommerce', etc. who would be best to help me come up with these different lists?